docs/quick_tutorial/tutorial_approach.rst

```diff
@@ -15,7 +15,7 @@ This "Getting Started" tutorial is broken into independent steps, starting with
 the smallest possible "single file WSGI app" example. Each of these steps
 introduces a topic and a very small set of concepts via working code. The steps
 each correspond to a directory in our workspace, where each step's directory is
-a Python package.
+a Python package. Source code used in this tutorial is located in the `Pyramid repository in the directory "docs/quick_tutorial" <https://github.com/Pylons/pyramid/tree/master/docs/quick_tutorial>`_. You may ``git clone`` the repository, download, or copy-paste the source code. If you do so, then make sure you use the same branch as this documentation.
```

Comment:
Sorry, I messed up. I should *not* have put a branch (`master`) in the URL because that would mean updating it for each release branch. Instead please use the URL `https://github.com/Pylons/pyramid/`. Then the subsequent two sentences clarify my intent.

Also please update the other 3 tutorials to reflect similar intent. Thank you!